_data-prepper/trace-analytics.md

```diff
@@ -0,0 +1,377 @@
+        # Fine-Grained Access Control, enable these settings. Comment out the
+        # username and password above.
+        #aws_sigv4: true
+        #aws_region: us-east-1
+  sink:
+    - opensearch:
+        hosts: [ "https://localhost:9200" ]
+        index_type: trace-analytics-raw
+        # Change to your credentials
+        username: "admin"
+        password: "admin"
+        # Add a certificate file if you are accessing an OpenSearch cluster with a self-signed certificate  
+        #cert: /path/to/cert
+        # If you are connecting to an Amazon OpenSearch Service domain without
+        # Fine-Grained Access Control, enable these settings. Comment out the
+        # username and password above.
+        #aws_sigv4: true
+        #aws_region: us-east-1
+service-map-pipeline:
+  workers: 8
+  delay: "100"
+  source:
+    pipeline:
+      name: "entry-pipeline"
+  processor:
+    - service_map_stateful:
+        # The window duration is the maximum length of time the data prepper stores the most recent trace data to evaluvate service-map relationships. 
+        # The default is 3 minutes, this means we can detect relationships between services from spans reported in last 3 minutes.
+        # Set higher value if your applications have higher latency. 
+        window_duration: 180 
+  buffer:
+      bounded_blocking:
+         # buffer_size is the number of ExportTraceRequest from otel-collector the data prepper should hold in memeory. 
+         # We recommend to keep the same buffer_size for all pipelines. 
+         # Make sure you configure sufficient heap
+         # default value is 512
+         buffer_size: 512
+         # This is the maximum number of request each worker thread will process within the delay.
+         # Default is 8.
+         # Make sure buffer_size >= workers * batch_size
+         batch_size: 8
+  sink:
+    - opensearch:
+        hosts: [ "https://localhost:9200" ]
+        index_type: trace-analytics-service-map
+        # Change to your credentials
+        username: "admin"
+        password: "admin"
+        # Add a certificate file if you are accessing an OpenSearch cluster with a self-signed certificate  
+        #cert: /path/to/cert
+        # If you are connecting to an Amazon OpenSearch Service domain without
+        # Fine-Grained Access Control, enable these settings. Comment out the
+        # username and password above.
+        #aws_sigv4: true
+        #aws_region: us-east-1
+```
+
+You need to modify the preceding configuration for your OpenSearch cluster. Note that it has two `opensearch` sinks which need to be modified.
+{: .note}
+
+You must make the following changes:
+* `hosts` - Set to your hosts
+* `username` - Provide the OpenSearch username.
+* `password` - Provide your OpenSearch password.
+* `aws_sigv4` - If you are using Amazon OpenSearch Service with AWS signing, set this value to `true`. It will sign requests with the default AWS credentials provider.
```

Comment:
```suggestion
* `aws_sigv4` – If you are using Amazon OpenSearch Service with AWS signing, set this value to `true`. It will sign requests with the default AWS credentials provider.
```